### PR TITLE
Security img metadata

### PR DESCRIPTION
**Pull request by ezrajrice**
Fix for "Metadata Information Leakage". Removed EXIF, IPTC, and XMP metadata from png and jpg files to address Issue #630

### CODE REVIEW

**Comment by ezrajrice** [comment]
@swainn Can you help me understand why this has so many commits? It's been a while since I created a PR and I can't remember the official process and I couldn't find any documentation on it. 

Steps:
1. Pulled the latest upstream/master
2. Rebased my local dev branch with upstream/master
3. Created new branch "security_img_metadata" from local dev
4. Made changes to only png and jpg files on "security_img_metadata"
5. Pushed changes to my fork of Tethys
6. Created Pull Request Draft

For some reason those steps created a bunch of commits that are being included in the PR.

**Comment by swainn** [comment]
@ezrajrice it looks like you branched off of an old feature branch. We squash merge PRs, so that all the commits get merged into one commit. As a result, the commits on the branch that features are developed in won't match the master branch. If you try to rebase the feature branch you'll have to deal with a ton of conflicts b/c every change is effectively contained in two commits: the squashed commit and one of the original commits. You should always delete the feature branches after they get merged and start new branches from master to avoid confusion.

TLDR; You need to check out a fresh copy of master, create a new branch, and then cherry-pick  your changes onto that branch.
